Description: Produce more verbose test results
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Abstract:
 This patch has been added for debugging Debian bug #879937.
 .
 It will be removed in a later upload once the issue is resolved.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -62,7 +62,7 @@
 
 ADD_CUSTOM_TARGET(
 	check
-	${CMAKE_CTEST_COMMAND} --force-new-ctest-process --output-on-failure
+	QDBUS_DEBUG=1 ${CMAKE_CTEST_COMMAND} --force-new-ctest-process -VV
 )
 
 find_package(CoverageReport)
